units/commit_queue.go

```diff
@@ -161,6 +189,22 @@ func (j *commitQueueJob) Run(ctx context.Context) {
 	})
 }
 
+func IsPatchDone(patch_id string) (bool, error) {
```

Comment:
We use camelcase rather than underscores by convention  (i.e. `patchId`)

Reply:
https://golang.org/doc/effective_go.html#mixed-caps